test/tools/api.js

```diff
@@ -111,6 +111,8 @@ const asyncApi = [
 
   { className: 'MongoClient', method: 'close', returnType: 'Promise<void>' },
   { className: 'MongoClient', method: 'connect', returnType: 'Promise<this>' },
+  // Manually test the static version of connect
+  // { className: 'MongoClient', method: 'static connect', returnType: 'Promise<this>' },
```

Comment:
can you add more context on how this is intended to be used or not used? i.e., is this ever intended to be uncommented? (something like "this is listed here as a reference for completeness, but we actually test it in a separate block in <test file>")

Reply:
sure, added a comment

Reply:
updated!

Reply:
can we add similar info in L43 in test/tools/api.js?